.github/workflows/stale.yml

```diff
@@ -0,0 +1,74 @@
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+name: "Close stale PRs"
+on:
+  schedule:
+    - cron: "30 1 * * 1" # Run once per week on Monday at 1:30 AM UTC
+
+jobs:
+  close-stale-prs:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          stale-pr-message: "Thank you for your contribution. Unfortunately, this pull request has been marked as stale because it has had no activity in the past 365 days. Please remove the stale label or comment below, or this PR will be closed in 14 days.  Feel free to re-open this if it has been closed in error. If you do not have repo permissions to reopen the PR, please tag a relevant maintainer."
+          stale-pr-label: "Status: stale-warning"
+          days-before-pr-stale: 365
+          days-before-pr-close: 14
+          # exclude issues
+          days-before-issue-stale: -1
+          days-before-issue-close: -1
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+  close-stale-issues-usage:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # exclude PRs
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          only-issue-labels: "Type: usage"
+          stale-issue-message: "This issue has been marked as stale because it has had no activity in the past 365 days. Please remove the stale label or comment below, or this issue will be closed in 14 days. If this usage question has evolved into a feature request or docs update, please remove the 'Type: usage' label and add the 'Type: enhancement' label instead."
+          stale-issue-label: "Status: stale-warning"
+          days-before-issue-stale: 365
+          days-before-issue-close: 14
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+  close-stale-issues-enhancement:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v9
+        with:
+          # exclude PRs
+          days-before-pr-stale: -1
+          days-before-pr-close: -1
+          only-issue-labels: "Type: enhancement"
+          exempt-issue-labels: "Status: needs champion"
+          stale-issue-message: "This issue has been marked as stale because it has had no activity in the past 365 days. Please remove the stale label or comment below, or this issue will be closed in 14 days. If this improvement is still desired but has no current owner, please add the 'Status: needs champion' label."
+          stale-issue-label: "Status: stale-warning"
+          days-before-issue-stale: 365
+          days-before-issue-close: 14
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
Rationale here is again about improving signal-to-noise ratio and so we can better identify enhancements to prioritise as opposed to tangential ideas which are unlikely to be completed and there is no real interest in anyway.